depends/patches/zeromq/fix_dash_compat.patch

```diff
@@ -0,0 +1,71 @@
+commit 058158724b3003fa348a338a5b8f1d058dc59b99
```

Comment:
Cannot find a commit with this hash.
Maybe https://github.com/zeromq/libzmq/commit/9a397666d28ca5f3c0d8233be3d39b2206555f45?